src/app/create-report/create-report.component.ts

```diff
@@ -254,7 +260,16 @@ export class CreateReportComponent implements OnInit, AfterViewInit {
     { displayText: DateFilterName.YESTERDAY, numDays: 1 },
     { displayText: DateFilterName.LAST_TWO_DAYS, numDays: 2 },
     { displayText: DateFilterName.LAST_WEEK, numDays: 7 },
+    { displayText: DateFilterName.LAST_MONTH, numDays: 31 },
+    { displayText: DateFilterName.CUSTOM, customOption: true },
```

Comment:
Rename this option to `fullDateDropdownOptions` and the other to `essentialDateDropdownOptions`.

Then
`dateDropdownOptions = this.useEssentialOrElevatedV2 ?  essentialDateDropdownOptions : fullDateDropdownOptions`